### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by rndmh3ro**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -38,9 +38,9 @@ jobs:
         uses: github-actions-x/commit@v2.9
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          push-branch: 'master'
-          commit-message: 'update changelog'
-          force-add: 'true'
+          push-branch: master
+          commit-message: update changelog
+          force-add: "true"
```
Comment (schurzi) [inline .github/workflows/release.yml:43]: are quotes here neccessary?

File: roles/icinga_downtime/defaults/main.yml [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
-comment: 'Downtime set by automation'
-host_downtime: 'true'
-service_downtime: 'true'
-single_service_downtime: 'false'
-single_host_downtime: 'false'
+comment: Downtime set by automation
+host_downtime: "true"
+service_downtime: "true"
+single_service_downtime: "false"
+single_host_downtime: "false"
```
Comment (schurzi) [inline roles/icinga_downtime/defaults/main.yml:9]: aren't this also bools?

File: roles/icinga_downtime/tasks/add_downtimes.yml [hunk 3]
```diff
@@ -1,82 +1,65 @@
...
-             "author": "{{ author }}",
-             "comment": "{{ comment }}",
-             "start_time": "{{ start_time_now }}",
-             "end_time": "{{ end_time }}",
-             "pretty": true,
-             "fixed": true,
-           }'
+    body: '{ "filter": "host.name==\"{{ item }}\"", "author": "{{ author }}", "comment":
+      "{{ comment }}", "start_time": "{{ start_time_now }}", "end_time": "{{ end_time
+      }}", "pretty": true, "fixed": true, }'
```
Comment (schurzi) [inline roles/icinga_downtime/tasks/add_downtimes.yml:16]: this feelds a bit like our recent JSON/YAML discussion. I like the structured layout with one key per line better

same below

File: .github/workflows/release.yml (hunk 1 above)
Comment (rndmh3ro) [reply .github/workflows/release.yml]: I changed all occurences and usages to bools.

File: roles/icinga_downtime/tasks/add_downtimes.yml (hunk 3 above)
Comment (rndmh3ro) [reply roles/icinga_downtime/tasks/add_downtimes.yml]: Yeah, reverted.

**Review by schurzi — APPROVED** [review]

**Review by xFuture603 — APPROVED** [review]

**Pull request merged**